### DISCUSSION TITLE
RadialMean Function Cutoff

### DISCUSSION THREAD

**Discussion by hwismer** [DIPlib/diplib#124] [General] [answered]
Hi All,

I'm using the diplib RadialMean() function on some image data and had a question about the implementation. I see that in the documentation you can either set the maxRadius to "outer radius" (largest distance from center to an edge pixel) or "inner radius" (shortest distance from center to edge pixel).

I'm using the "outer radius" at the moment and was curious how values outside the image are handled. For example, if the center point is placed towards one corner of the image are positions outside of the image perhaps treated as zeros or simply ignored?

Thanks!

**Comment by crisluengo** [comment, marked as answer]
It computes the mean of pixels in the image only, the area outside the image is ignored.

**Reply by hwismer** [reply]
Perfect. Good to know. Thank you!
